pom.xml

```diff
@@ -285,49 +297,49 @@
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
-                <artifactId>maven-dependency-plugin</artifactId>
-                <version>${maven.dependency-plugin.version}</version>
-                <inherited>false</inherited>
+                <artifactId>maven-antrun-plugin</artifactId>
                 <executions>
                     <execution>
-                        <id>download-test-logs</id>
                         <phase>process-test-resources</phase>
                         <goals>
-                            <goal>unpack</goal>
+                            <goal>run</goal>
                         </goals>
                         <configuration>
-                            <skip>${skipUnpack}</skip>
-                            <artifactItems>
-                                <artifactItem>
-                                    <groupId>com.microsoft.gctoolkit</groupId>
-                                    <artifactId>gctoolkit-gclogs</artifactId>
-                                    <version>${gctoolkit-testdata-version}</version>
-                                    <type>zip</type>
-                                </artifactItem>
-                                <artifactItem>
-                                    <groupId>com.microsoft.gctoolkit</groupId>
-                                    <artifactId>gctoolkit-gclogs-rolling</artifactId>
-                                    <version>${gctoolkit-testdata-version}</version>
-                                    <type>zip</type>
-                                </artifactItem>
-                                <artifactItem>
-                                    <groupId>com.microsoft.gctoolkit</groupId>
-                                    <artifactId>gctoolkit-shenandoah-logs</artifactId>
-                                    <version>${gctoolkit-testdata-version}</version>
-                                    <type>zip</type>
-                                </artifactItem>
-                                <artifactItem>
-                                    <groupId>com.microsoft.gctoolkit</groupId>
-                                    <artifactId>gctoolkit-zgc-logs</artifactId>
-                                    <version>${gctoolkit-testdata-version}</version>
-                                    <type>zip</type>
-                                </artifactItem>
-                            </artifactItems>
-                            <includes>**/*</includes>
-                            <outputDirectory>${project.basedir}/gclogs
-                            </outputDirectory>
+                            <target>
+                                <get skipexisting="true" src="https://github.com/microsoft/gctoolkit-testdata/archive/refs/tags/${gctoolkit-testdata-version}.zip" dest="${maven-parent.basedir}"/>
```

Comment:
This is the src zip file from the release. The source uses LFS and since src.zip is just a simple zip of the source, the large file links are not resolved. Need to look into creating a release where src.zip has LFS links resolved.

Reply:
Seems relevant: https://stackoverflow.com/questions/65110940/zipping-git-lfs-binaries-in-github-actions-and-adding-them-as-asset-to-a-release

Reply:
Changes are in the main branch on my fork. PR coming today.

https://github.com/brunoborges/gctoolkit-testdata/blob/main/.github/workflows/release.yml